graphql-compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/GraphqlFunctionValidator.java

```diff
@@ -98,6 +102,19 @@ private void validateResourceAccessorName(MethodSymbol methodSymbol, Location lo
         }
     }
 
+    private void validateResourcePath(MethodSymbol methodSymbol, Location location, SyntaxNodeAnalysisContext context) {
+        if (methodSymbol.kind() == SymbolKind.RESOURCE_METHOD) {
+            ResourceMethodSymbol resourceMethodSymbol = (ResourceMethodSymbol) methodSymbol;
+            if (resourceMethodSymbol.resourcePath().signature().isEmpty()) {
+                PluginUtils.updateContext(context, CompilationErrors.INVALID_RESOURCE_PATH, location);
```

Comment:
What does this condition mean?

Reply:
I don't think we'll ever come across a scenario like this. A resource path will never be empty. If it is, that will be captured in the compiler itself.